src/components/core/PivotTable.tsx

```diff
@@ -967,6 +967,29 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private getInfiniteInitialRowCountRowCount() {
+        // for init load this method return page after sort change return last execution rowcount
+        // this value is used in gridOption and remove it unnecessary horizontal scrollbars flickering when
+        // new execution is done after sort change
+        const { execution } = this.state;
+        const { pageSize } = this.props;
+        let rowCount = 0;
+
+        if (
+            execution &&
+            execution.executionResult &&
```

Comment:
replace this by this.getExecutionResult()

Reply:
Done